readme.md

```diff
@@ -38,63 +38,65 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** Forking is a request to clone and registers under your username.
 
 ---
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
-* **Your Answer:**
+* **Your Answer:** Created a package.json file with information about the author, repo, etc.
```

Comment:
Additionally, the `-y` flag responds "Yes" to all questions that would've been asked by `npm init`. Just try `npm init` out on its own to see what would happen without the `-y` flag.